### PR TITLE
update examples

### PR DESCRIPTION
**Pull request by mmckane**

### CODE REVIEW

File: workflow-templates/k8s-deploy-prod.yml [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
     branches: [ $default-branch ]
 
 jobs:
-  # Builds your container image
+  # Builds and promote your container image
   build-container:
     uses: gmi-actions/build-container/.github/workflows/build-container.yml@v1
     with:
       container_name: ${{ github.event.repository.name }} # Defaults container name to your repository name
+      artifactory_promotion_branch: refs/heads/$default-branch # Set Promotion Branch to this repositories default branch
```
Comment (jwillaz) [inline workflow-templates/k8s-deploy-prod.yml:13]: Consider changing to "artifact_promotion_branch". We are going to genericize the artifact actions, as we may employ multiple registries in the future.

**Review by jwillaz — APPROVED** [review]

File: workflow-templates/k8s-deploy-prod.yml (hunk 1 above)
Comment (mmckane) [reply workflow-templates/k8s-deploy-prod.yml:13]: this code is specific to the build-container action and releveant only to artifactory as it runs the jfrog CLI.

File: workflow-templates/k8s-pr-deploy-nonprod.yml [hunk 2]
```diff
@@ -37,11 +47,54 @@ jobs:
...
-  sync:
-    if: github.ref != 'refs/heads/$default-branch'
+  dev-sync:
+    # Ignore dependabot PR's note only works from on: pull_request events. Switch to github.ref_name for pushes.
+    if: (!startsWith(github.head_ref, 'dependabot'))
     uses: gmi-actions/deployK8sApplication/.github/workflows/sync.yml@v1
-    needs: deploy
+    needs: dev-deploy
     with:
       matrix: ${{ needs.deploy.outputs.clusterMatrix }}
```
Comment (mmckane) [inline workflow-templates/k8s-pr-deploy-nonprod.yml:56]:
```suggestion
      matrix: ${{ needs.dev-deploy.outputs.clusterMatrix }}

```

File: workflow-templates/k8s-pr-deploy-nonprod.yml [hunk 3]
```diff
@@ -37,11 +47,54 @@ jobs:
...
+  #             - ${{ github.event.repository.name }}-${{vars.env_name}}.k8s.genmills.com
+
+  # # Deploys your application to the cluster(S) by asking argocd in the cluster to syncronize with your git repository
+  # sync:
+  #   # Ignore dependabot PR's note only works from on: pull_request events. Switch to github.ref_name for pushes.
+  #   if: (!startsWith(github.head_ref, 'dependabot'))
+  #   uses: gmi-actions/deployK8sApplication/.github/workflows/sync.yml@v1
+  #   needs: qa-deploy
+  #   with:
+  #     matrix: ${{ needs.deploy.outputs.clusterMatrix }}
```
Comment (mmckane) [inline workflow-templates/k8s-pr-deploy-nonprod.yml:98]:
```suggestion
  #     matrix: ${{ needs.qa-deploy.outputs.clusterMatrix }}

```

**Pull request merged**